workflows/files/ZWECONF.xml

```diff
@@ -509,6 +509,28 @@ How we want to verify SSL certificates of services. Valid values are:
             <default>IZUDFLT</default>
         </string>
     </variable>
+    <variable name="components_apiml_enabled" scope="instance" visibility="public">
```

Comment:
Should this be removed? Along with the `ZWECONF.properties` updates? I saw you marked these as missing in the sync rules, so if you want to keep them defaulted in both ZWECONF and ZWEAMLCF, removing the variable definitions would clarify that.

Reply:
I guess the question to ask is: do we want users to see the modulith is even an option in 3.3? Or is the intent to keep this more hidden and work with users directly who might benefit from trying out the modulith?